examples/evaluate_different_templates_num_demos.py

```diff
@@ -0,0 +1,30 @@
+import evaluate
+from datasets import load_dataset
+from unitxt.text_utils import print_dict
+
+# Use the HF load_dataset API, to load the wnli entailment dataset using the standard template in the catalog for relation task with 5-shot in-context learning.
+# We set loader_limit to 200 to limit reduce download time.
+# Sample for each example a template from [templates.classification.multi_class.relation.default,templates.key_val]
+# Sample for each example a number of demonstrations from [0,5]
+dataset = load_dataset(
+    "unitxt/data",
+    "card=cards.wnli,template=[templates.classification.multi_class.relation.default,templates.key_val],num_demos=[0,5],demos_pool_size=100,loader_limit=200",
+    trust_remote_code=True,
+)
+
+# Print the resulting dataset.
```

Comment:
I think that instead of showing how to evaluate again, you can print out the different instances and the selected template / num demos.